modules/textures/src/basis-loader.ts

```diff
@@ -18,7 +18,9 @@ export const BasisWorkerLoader = {
   options: {
     basis: {
       format: 'rgb565', // TODO: auto...
-      libraryPath: 'libs/'
+      libraryPath: 'libs/',
+      decoderFormat: 'Basis', // 'Basis' || 'KTX2'
```

Comment:
I recommend lower case strings for all option values. `'basis', 'ktx2'`

Reply:
Done